target/linux/realtek/dts/rtl9302_plasmacloud_psx8.dtsi

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "rtl930x.dtsi"
+
```

Comment:
Please rename this DTS include to rtl9302_plasmacloud.dtsi or rtl9302_plasmacloud_common.dtsi. While it is basically the whole PSX8 file we never use dedicated model names in dtsi files. Instead provide a short and clean rtl9302_plasmacloud_psx8.dts (without the I) derived from that (like the PSX10).